c10/core/thread_pool.cpp

```diff
@@ -44,6 +45,7 @@ ThreadPool::ThreadPool(
       if (init_thread) {
         init_thread();
       }
+      c10::setThreadName("pt_thread_pool");
```

Comment:
We should probably run this before the user provided init_thread function so it can override the name if necessary. Thoughts?

Reply:
Good point! Lemme change it.